base.py

```diff
@@ -1,20 +1,29 @@
 from abc import ABC, abstractmethod
 
-import joblib
 from joblib import delayed, effective_n_jobs, Parallel
-from typing import List, Union
+from typing import List, Optional, Union
 
 import numpy as np
 import pandas as pd
+import scipy.sparse as spsparse
+
 from sklearn.base import BaseEstimator, TransformerMixin
 
 from rdkit.Chem import MolFromSmiles
 from rdkit.Chem.rdchem import Mol
 
 
 class FingerprintTransformer(ABC, TransformerMixin, BaseEstimator):
-    def __init__(self, n_jobs: int = None):
+    def __init__(
+        self,
+        n_jobs: Optional[int] = None,
+        sparse: bool = False,
+        count: bool = False,
+    ):
         self.n_jobs = effective_n_jobs(n_jobs)
+        self.sparse = sparse
+        self.count = count
+        self.fp_generator_kwargs = {}
```

Comment:
Remove (reasons outlined below in Morgan fingerprint)

Reply:
I've removed the fp_generator_kwargs argument. Why should we also remove sparse and count arguments?

Reply:
Sorry, my mistake - `sparse` and `count` should stay